provider/core_security_list_resource_test.go

```diff
@@ -158,10 +242,19 @@ func (s *ResourceCoreSecurityListTestSuite) TestAccResourceCoreSecurityList_basi
 						egress_security_rules = []
 						ingress_security_rules = []
 					}
+
+					resource "oci_core_security_list" "default" {
+						manage_default_resource_id = "${oci_core_virtual_network.t.default_security_list_id}"
+						display_name = "default-tf-security_list-updated"
+						egress_security_rules = []
+						ingress_security_rules = []
+					}
 				`,
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(s.ResourceName, "egress_security_rules.#", "0"),
 					resource.TestCheckResourceAttr(s.ResourceName, "ingress_security_rules.#", "0"),
+					resource.TestCheckResourceAttr(s.DefaultResourceName, "egress_security_rules.#", "0"),
+					resource.TestCheckResourceAttr(s.DefaultResourceName, "ingress_security_rules.#", "0"),
```

Comment:
How about testing that a default resource can be applied, then removed from the config and applied, then added back and applied?

Reply:
This is a good idea for a test case. I will add this.